packages/mermaid/src/rendering-util/types.d.ts

```diff
@@ -0,0 +1,7 @@
+export type MarkdownWordType = 'normal' | 'strong' | 'emphasis';
+export interface MarkdownWord {
+  content: string;
+  type: MarkdownWordType;
+}
+export type MarkdownLine = MarkdownWord[];
+export type CheckFitFunction = (text: MarkdownLine) => boolean;
```

Comment:
It might be worth documenting whether this function returns `true` if it fits, or whether it returns `false` if it fits.

```suggestion
/** Returns `true` if the line fits a constraint (e.g. it's under 𝑛 chars) */
export type CheckFitFunction = (text: MarkdownLine) => boolean;
```